protocols/src/N2NP.nroff

```diff
@@ -462,21 +462,22 @@ message, but rather this payload:
 └───────┴─────────────────────────────────────────────────────────────────────┘
 .in 2
 
-The Channel ID, on 2 bytes, describes the ID of an channel than a client want
-to ass his message to. This ID is handled with the CRT message mechanic, see
-the appropriate section for more information. This packet is, by essence, a
+The Channel ID, 2 bytes in legth, describes the ID of a channel that a client
+wants to pass his message to. This ID is handled with the CRT message mechanic,
+see the appropriate section for more information. This packet is, in essence, a
 broadcast; The destination device ID should be set to 0. Every client must
-relay this message to their neighbors, and log it internally. This way, if a 
-client shall join a channel, it will have history (at least from the moment 
-of he joined the network), and when this client wants to join / leave a channel
-he will be doing it internally, without announcing it to the all network.
+relay this message to their neighbors, and log it internally. This way, when a 
+client joins a channel, it will have a history (at least from the moment 
+he joined the network). When a client wants to join / leave a channel he will
+be doing this internally, without announcing it to the network.
```

Comment:
```
[... ] join / leave a channel, he will be doing this internally [...]
                               ^  ^
```
Corrected:
```
[... ] join / leave a channel, it needs to be doing this internally [...]
```

Reply:
Since the whole document already uses `he` to address a node, changing that to it for this one case would only cause confusion

I agree that it fits better, but I ignored that for now for time reasons and intended to make another PR in the future to fix this if it hasn't been done when I have some time for it

My changes on this were merely reformatting to not make it look bad when built

Reply:
@nepcore no problems
My comment is more of a "hey look, there's also this".

For the formating, `groff` handles it automatically; each paragraph could be a single line in the source. Optimally, it should be in one line, since chopping them in the source bypass `groff`'s word-wrap.

Reply:
I have never used `groff`, i didn't know about this, maybe someone should actually go through and fix that then ;)

Reply:
That's long-term, and there is a bit of resistance against that in the name of "readability". IMO, it's moot since it's a source file, but hey ¯\_(ツ)_/¯